packages/eui/src/components/selectable/selectable_list/selectable_list.stories.tsx

```diff
@@ -107,3 +111,34 @@ export const Playground: Story = {
     },
   },
 };
+
+export const Groups: Story = {
+  parameters: {
+    controls: {
+      include: ['options'],
+    },
+  },
+  args: {
+    options: [
+      { label: 'Group 1', isGroupLabel: true },
+      ...[...options].splice(0, 4),
+      {
+        label: 'Group 2',
+        isGroupLabel: true,
+        prepend: <EuiIcon type="warning" />,
```

Comment:
Very optional, feel free to ignore or skip this - some extra spacing after this icon might look nice!
```suggestion
        prepend: <EuiIcon type="warning" css={({ euiTheme }) => ({ marginRight: euiTheme.size.s })} />,
```

Reply:
Ah yes, I agree to add this considering we showcase currently expected usage 👍

Reply:
Updated in this [commit](https://github.com/elastic/eui/pull/7800/commits/b852f709ae24c57237a09c73d372634438b5e94d)